packages/scene-composer/src/components/panels/scene-settings/SceneTagSettingsEditor.tsx

```diff
@@ -19,13 +19,19 @@ export const SceneTagSettingsEditor: React.FC = () => {
   const isViewing = useStore(sceneComposerId)((state) => state.isViewing());
   const setTagSettings = useViewOptionState(sceneComposerId).setTagSettings;
   const tagSettings: ITagSettings = useTagSettings();
-  const [dirty, setDirty] = useState(false);
   const [focusInput, setFocusInput] = useState(false);
   const [focusSlider, setFocusSlider] = useState(false);
   const [draggingSlider, setDraggingSlider] = useState(false);
-
   const [internalScale, setInternalScale] = useState(tagSettings.scale);
 
+  const handleInputFocus = useCallback(() => setFocusInput(true), []);
+  const handleInputBlur = useCallback(() => setFocusInput(false), []);
+
+  const handleSliderFocus = useCallback(() => setFocusSlider(true), []);
```

Comment:
Unfortunately we need all of these state managers. `mouseUp` & `mouseDown` are required in order for the slider to work & `focus`/`blur` are necessary for each independently as well.